### PR TITLE
🐛 quick fix to get all occurrences for oncogrid

### PR DESCRIPTION
**Pull request by Jephuff**
filter by case_ids and some simple chunking if it's over 1000

### CODE REVIEW

File: modules/node_modules/@ncigdc/components/Oncogrid/getQueries.js [hunk 1]
```diff
@@ -38,34 +36,44 @@ async function getGenes({
...
+    consequenceTypes,
+    impacts,
+    currentFilters,
+    from = 0,
+    hits = [],
+  } = args;
+
+  if (!geneIds.length || !caseIds.length) return { data: { hits } };
+  if (from + OCCURRENCE_CHUNK > 10000) {
+    console.warn('to many occurrences to fetch');
```
Comment (alex-wilmer) [inline modules/node_modules/@ncigdc/components/Oncogrid/getQueries.js]: "too many"

**Comment by alex-wilmer** [comment]
This seems reasonable. What's a good way to test it?

File: modules/node_modules/@ncigdc/components/Oncogrid/getQueries.js (hunk 1 above)
Comment (Jephuff) [inline modules/node_modules/@ncigdc/components/Oncogrid/getQueries.js]: oops!

**Comment by Jephuff** [comment]
Good question, only test I can think of is to make sure the 50 genes make sense (before this they were getting filtered in the front end so it wasn't showing 50 genes)

**Comment by shanewilson** [comment]
status?

**Comment by Jephuff** [comment]
needs review

**Comment by alex-wilmer** [comment]
I've counted a number of times and I'm getting 49

**Comment by alex-wilmer** [comment]
image

**Comment by Jephuff** [comment]
hmm exploration?

**Comment by alex-wilmer** [comment]
yes with no filters. should be I checking a different page?

**Comment by Jephuff** [comment]
nope, just want to know where to debug :D

**Comment by alex-wilmer** [comment]
I changed it from `size=50` to 51 and I get 50 now

**Comment by shanewilson** [comment]
might be a bug in API

**Comment by Jephuff** [comment]
ahh, I wasn't thinking. before this change it was dropping genes because the occurrences were truncated, now it's dropping them because the cases are truncated.

**Comment by alex-wilmer** [comment]
> ahh, I wasn't thinking

I doubt it. Sooo not a UI issue then?

**Comment by Jephuff** [comment]
yep, not a ui issue

**Pull request merged**